examples/directml/stable_diffusion/config_safety_checker.json

```diff
@@ -56,7 +56,27 @@
                 "float16": true,
                 "use_gpu": true,
                 "keep_io_types": false,
-                "target_provider": "directml_future"
+                "optimization_options": {
```

Comment:
Whatever is in the `examples` folder is not part of the olive package, right? Do users really have to manually set which fusions they want and which ones they don't want? It feels like we should be deciding which fusions are appropriate for DML.

Reply:
Correct, examples are not published as part of olive-ai package.

I agree with you that Olive should ideally decide fusions in a more automatic fashion, but the logic I added to OrtTransformerOptimizer is a bit dubious long term: it relies on a special "target_provider" parameter that will be obsolete with some upcoming changes (e.g. see #255). I want to revisit this in the future when accelerators are integrated.